Assignments/MYSQL/HRUDAY/Assignment_2.sql

```diff
@@ -0,0 +1,52 @@
+-- 1) Create a table course with the following details courseId number courseName varchar(20)....
+create table course(
```

Comment:
@Erhruday use caps for the keywords

Reply:
It's okk...

i'll definitely try to do it in next assignments......